tests/platform_tests/broadcom/test_ser.py

```diff
@@ -41,8 +41,7 @@ def enable_ssh_timout(dut):
     @param dut: Ansible host DUT
     '''
     logger.info('Enabling ssh time out on dut: %s' % dut.hostname)
-    dut.command("sudo sed -i '/^#ClientAliveInterval/s/^#//' /etc/ssh/sshd_config")
-    dut.command("sudo sed -i '/^#ClientAliveCountMax/s/^#//' /etc/ssh/sshd_config")
+    dut.command("sudo cp /etc/ssh/sshd_config.bak /etc/ssh/sshd_config")
```

Comment:
Suggest also removing the backup file after (or using `mv` in lieu of `cp`)

Reply:
Sure.